src/streaming/MediaPlayer.js

```diff
@@ -1920,6 +1920,33 @@ function MediaPlayer() {
     }
 
     /**
+     * Get the value of useDeadTimeLatency in AbrController. @see setUseDeadTimeLatencyForAbr
+     *
+     * @returns {boolean=}
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function getUseDeadTimeLatencyForAbr() {
+        return abrController.getUseDeadTimeLatency();
+    }
+
+    /**
+     * Set the value of useDeadTimeLatency in AbrController. If true, only the download
+     * portion will be considered part of the download bitrate and latency will be
+     * regarded as static. If false, the reciprocal of the whole transfer time will be used.
+     * Defaults to true.
+     *
+     * @param {boolean=} useDeadTimeLatency - True or false flag.
+     *
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function setUseDeadTimeLatencyForAbr(useDeadTimeLatency) {
+        abrController.setUseDeadTimeLatency(useDeadTimeLatency);
+    }
```

Comment:
Regarding API breaking, we could just proxy those calls to set the relevant config and mark the functions as deprecated for now; but if anyone is relying on this reset behaviour, we would be breaking that (hopefully nobody, perhaps regard that as an acceptable risk).